Dockerfile

```diff
@@ -35,11 +43,13 @@ RUN poetry install --without dev --no-root && rm -rf $POETRY_CACHE_DIR
 RUN make compilemessages
 
 # The runtime image, used to just run the code provided its virtual environment
-FROM python:3.11-slim-buster AS runtime
+FROM ${ecr_path}${python_version}-${alpine_version} AS runtime
+
+# Workaround for CVE-2024-6345 upgrade the installed version of setuptools to the latest version
+RUN pip install -U setuptools
```

Comment:
will we be able to remove this when we next update the base image?

Reply:
A later python version would fix it now.